modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java

```diff
@@ -759,11 +764,31 @@ public Response addMediaPackage(@Context HttpServletRequest request, @PathParam(
       int episodeDCCatalogNumber = 0;
       boolean hasMedia = false;
       if (ServletFileUpload.isMultipartContent(request)) {
+        // Validate first
         for (FileItemIterator iter = new ServletFileUpload().getItemIterator(request); iter.hasNext();) {
           FileItemStream item = iter.next();
           if (item.isFormField()) {
             String fieldName = item.getFieldName();
-            String value = Streams.asString(item.openStream(), "UTF-8");
+            String value = Streams.asString(item.openStream(), StandardCharsets.UTF_8.toString());
```

Comment:
Should this logic not also be added to `addZippedMediaPackage`?

Reply:
It does not have form params of type date, so no? Not that I'm against better param validation.